src/index.html

```diff
@@ -1,13 +1,332 @@
+                  <a href="#" class="nav-mobile__link">Home</a>
+                </li>
+                <li class="nav-mobile__item">
+                  <a href="#info" class="nav-mobile__link">About us</a>
+                </li>
+                <li class="nav-mobile__item">
+                  <a href="#gallery" class="nav-mobile__link">Gallery</a>
+                </li>
+                <li class="nav-mobile__item">
+                  <a href="#callback" class="nav-mobile__link">Consultation</a>
+                </li>
+                <li class="nav-mobile__item">
+                  <a href="#clients" class="nav-mobile__link">Testimonials</a>
+                </li>
+                <li class="nav-mobile__item">
+                  <a href="#contacts" class="nav-mobile__link">Contacts</a>
+                </li>
+              </ul>
+              <p class="nav-mobile__phone">+1 234 555-55-55</p>
+              <a href="tel:+1234567890" class="nav-mobile__call">Call to order</a>
+            </nav>
+          </div>
+
+          <h1 class="header__title">Miami<br> Condo Kings</h1>
+
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item nav__item--active">
+                <a href="#" class="nav__link">Home</a>
+              </li>
+              <li class="nav__item">
+                <a href="#info" class="nav__link">About us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#gallery" class="nav__link">Gallery</a>
+              </li>
+              <li class="nav__item">
+                <a href="#callback" class="nav__link">Consultation</a>
+              </li>
+              <li class="nav__item">
+                <a href="#clients" class="nav__link">Testimonials</a>
+              </li>
+              <li class="nav__item">
+                <a href="#contacts" class="nav__link">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+
+      </div>
+    </header>
+
+    <main class="container">
+
+      <section class="section info" id="info">
+        <h2 class="info__title">Miami best<br>Real estate <a href="https://goo.gl/maps/anCFi2RZg2mPREBo7">here</a></h2>
+        <article class="info__article">
+          <h3>Over 50 years experience in love.</h3>
+          <p>Mazel and VA Tsukkerman in his "Analysis of musical works."<br> Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          <p>The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information about the profitable deals.</p>
+          <p><strong>We will never be late!<br>Guaranteed secure transaction - 100%</strong></p>
```

Comment:
одна строка - один тег
```suggestion
          <p>
            <strong>
              We will never be late!<br>Guaranteed secure transaction - 100%
            </strong>
          </p>
```